org.adempiere.base/src/org/compiere/process/ProcessInfoParameter.java

```diff
@@ -282,4 +300,134 @@ public void setParameterName (String ParameterName)
 		m_ParameterName = ParameterName;
 	}
 
+	/**
+	 * Return the value of the parameter as a comma separated integer string. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public String getParameterAsMultipleInt() {
+		return getParameterMultipleInt(getParameterAsString());
+	}
+
+	/**
+	 * Return the value of the parameter To as a comma separated integer string. Validate every value is an integer and throws NumberFormatException if one of the value is not a valid integer.
+	 * @return String
+	 */
+	public String getParameter_ToAsMultipleInt() {
+		return getParameterMultipleInt(getParameter_ToAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as a validated String (all values between commas must be integer)
+	 * @return String
+	 */
+	private String getParameterMultipleInt(String param) {
+
+		if (Util.isEmpty(param))
+			return "";
+
+		String[] strarr = param.split(",");
+
+		for (String par : strarr)
+			Integer.valueOf(par);
+
+		return param;
+	}
+
+	/**
+	 * Return the value of the parameter as a String with all values between commas surrounded by quotes
+	 * @return String
+	 */
+	public String getParameterAsMultipleString() {
+		return getParameterMultipleString(getParameterAsString());
+	}
+
+	/**
+	 * Return the value of the parameter as a String with all values between commas surrounded by quotes
+	 * @return String
+	 */
+	public String getParameter_ToAsMultipleString() {

```

Comment:
I think a more direct and obvious naming is getParameterAsCSVString and getParameterAsCSVInt

Reply:
done